packages/use-dataloader/src/DataLoaderProvider.tsx

```diff
@@ -30,209 +33,186 @@ type UseDataLoaderInitializerArgs<T = unknown> = {
   maxDataLifetime?: number
 }
 
-interface Context {
-  addCachedData: (key: string, newData: unknown) => void
-  addReload: (key: string, method: () => Promise<void | unknown>) => void
+type GetCachedDataFn = {
+  (): CachedData
+  (key?: string): unknown | undefined
+}
+
+type GetReloadsFn = {
+  (): Reloads
+  (key?: string): (() => Promise<void | unknown>) | undefined
+}
+
+export interface IDataLoaderContext {
   addRequest: (key: string, args: UseDataLoaderInitializerArgs) => DataLoader
-  cacheKeyPrefix: string
+  cacheKeyPrefix?: string
   onError?: (error: Error) => void | Promise<void>
   clearAllCachedData: () => void
-  clearAllReloads: () => void
-  clearCachedData: (key?: string) => void
-  clearReload: (key?: string) => void
-  getCachedData: (key?: string) => unknown | CachedData
-  getReloads: (key?: string) => (() => Promise<void | unknown>) | Reloads
+  clearCachedData: (key: string) => void
+  clearRequest: (key: string) => void
+  getCachedData: GetCachedDataFn
+  getReloads: GetReloadsFn
   getRequest: (key: string) => DataLoader | undefined
   reload: (key?: string) => Promise<void>
   reloadAll: () => Promise<void>
 }
 
 // @ts-expect-error we force the context to undefined, should be corrected with default values
-export const DataLoaderContext = createContext<Context>(undefined)
+export const DataLoaderContext = createContext<IDataLoaderContext>(undefined)
 
 const DataLoaderProvider = ({
   children,
   cacheKeyPrefix,
   onError,
+  maxConcurrentRequests,
```

Comment:
```suggestion
  maxConcurrentRequests = DEFAULT_MAX_CONCURRENT_REQUESTS,
```
plutôt que les `defaultProps`

Reply:
Because of DataLoader class and Provider have a default value this will never happen

Reply:
Ok, it's set twice but with the same constant... not a big deal.